### PR TITLE
Create uwpx.doap

### PR DESCRIPTION
**Pull request by Echolon**
https://xmpp.org/extensions/xep-0453.html

Not sure if everything is correct - was close to midnight already :man_shrugging: 

FYI @wurstsalat3000

### CODE REVIEW

**Review by COM8 — APPROVED** [review]
Updated it.
Is there anything else we should add here before merging?

**Comment by COM8** [comment]
@wurstsalat3000, do you have any suggestions?

File: uwpx.doap [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+      <foaf:Person>
+        <foaf:name>Fabian Sauter</foaf:name>
+        <foaf:homepage rdf:resource="https://github.com/com8" />
+      </foaf:Person>
+    </maintainer>
+
+    <release>
+      <Version>
+        <revision>v.0.32.1.0</revision>
+        <created>20201-06-06</created>
```
Comment (wurstsalat3000) [inline uwpx.doap:197]: There is a typo in 20201

File: uwpx.doap [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+    <bug-database rdf:resource="https://github.com/UWPX/UWPX-Client/issues" />
+    <license rdf:resource="https://github.com/UWPX/UWPX-Client/blob/master/LICENSE" />
+    <schema:documentation rdf:resource="https://github.com/UWPX/UWPX-Client" />
+
+    <!-- See https://github.com/ewilderj/doap/issues/49 -->
+    <language>en</language>
+
+    <programming-language>C#</programming-language>
+    <os>Windows 10</os>
+    <os>UWP</os>
```
Comment (wurstsalat3000) [inline uwpx.doap:26]: I know this is technically correct, but maybe adding `<os>Windows</os>` would minimize confusion.

Comment (wurstsalat3000) [reply uwpx.doap:26]: Ah, strike that, I didn't see `Windows 10`.

File: uwpx.doap (hunk 1 above)
Comment (COM8) [reply uwpx.doap]: Thanks!

File: uwpx.doap (hunk 2 above)
Comment (COM8) [reply uwpx.doap:26]: Replaced it with `Windows`, since Windows 11 will be a thing 🙃.

**Pull request merged**